tau-component-packages/components/progress/package.json

```diff
@@ -81,6 +81,6 @@
       "thickness": "null"
     }
   },
-  "resizable": true,
+  "resizable": false,
```

Comment:
Why it need to set the resizable to false?
So we can't resize the progress widget?

Reply:
Now resizing crashes progress. I've found some additional issues, I will write it on GH, and fix 
![image](https://user-images.githubusercontent.com/18300603/65229847-37c02180-dacd-11e9-8b03-9ab51271e0ea.png)

Reply:
Issue #369 is about it. Until fix it, we decided to turn off resizing

Reply:
ok. next time, fix it please.